.github/CONTRIBUTING.md

```diff
@@ -96,7 +97,21 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirety.
+
+Any pull-request that does not adequately complete the provided template may be closed or marked 'do not merge' by maintainers.
+ - Balance changes must indicate that they affect the balance of the game.
+ - Changes must be documented in their entirety including the extent of their effects. (For example, if you change it so all mobs are half speed, don't label the PR as 'monkeys now move twice as slow'). Failing to document the full extent of the changes may result in a repo-ban if the intent of hiding changes is seen as malicious.
+ - The section labeled 'about this pull request' should state the changes that the pull-request makes.
+ - The section labeled "why it's good for the game" should include the reasons behind the changes and how they will be good for the game.
+ - The testing section should contain screenshots, videos, and/or reproducible testing procedures showing that the PR works as specified. Pull-requests that ignore this section, or are not tested, may be closed by maintainers. This applies to small PRs that may seem trivial.
+ - The changelog should include a short summary of the changes made; including any other authors that contributed to the pull-request (Include their names after the :cl: tag along with your own)
+
+If a pull-request requires updates to the wiki, these changes should be made on your user account page (For example: https://wiki.beestation13.com/view/User:Powerfulbacon), so that the original page can be updated on merge.
```

Comment:
```suggestion
If a pull-request requires updates to the wiki, these changes should be made on your user account page (For example: https://wiki.beestation13.com/view/User:PowerfulBacon), so that the original page can be updated on merge.
```